File: app/src/main/java/com/gee12/mytetroid/fragments/settings/SettingsHandlingFragment.java
package com.gee12.mytetroid.fragments.settings;

import android.os.Bundle;

import com.gee12.mytetroid.R;

public class SettingsHandlingFragment extends TetroidSettingsFragment {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        super.onCreatePreferences(savedInstanceState, rootKey);
        setPreferencesFromResource(R.xml.prefs_handling, rootKey);

        getActivity().setTitle(R.string.pref_category_handling);
    }
}

File: app/src/main/java/com/gee12/mytetroid/ui/fragments/settings/SettingsHandlingFragment.kt
package com.gee12.mytetroid.ui.fragments.settings

import android.os.Bundle
import com.gee12.mytetroid.R

class SettingsHandlingFragment : TetroidSettingsFragment() {

    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        super.onCreatePreferences(savedInstanceState, rootKey)
        setPreferencesFromResource(R.xml.prefs_handling, rootKey)
        requireActivity().setTitle(R.string.pref_category_handling)
    }
}